cybershell/vulnerability_kb.py

```
@@ -0,0 +1,740 @@
+                "confidence": payload.confidence_score,
+                "product_match": payload.product == target_info.get('product') if target_info else False
+            }
+            results.append(result)
+        
+        return results
+    
+    def _build_test_url(self, base_url: str, payload: VulnPayload) -> str:
+        """Build test URL with payload"""
+        if payload.context == "parameter":
+            return f"{base_url}?test={payload.payload}"
+        elif payload.context == "path":
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)